.github/workflows/release.yml

```diff
@@ -42,8 +45,35 @@ jobs:
           SNAPCRAFT_TOKEN: ${{ secrets.SNAPCRAFT_TOKEN }}
           SPLIT_PROD_KEY: ${{ secrets.SPLIT_PROD_KEY }}
           SPLIT_STAGING_KEY: ${{ secrets.SPLIT_STAGING_KEY }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         run: make release-publish
 
+      - uses: "finnp/create-file-action@master"
+        env:
+          FILE_NAME: "dist/currentVersion.txt"
+          FILE_DATA: "${{ steps.get-version.outputs.version }}"
+
+      - name: Upload currentVersion.txt to S3
+        id: upload-current-version-s3
+        uses: ItsKarma/aws-cli@v1.70.0
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_DEFAULT_REGION: "us-west-1"
```

Comment:
@ctrombley not that this is exactly a secret, what do you think about storing the region value in the secrets along side the access/secret keys?